frontend/src/components/Card.tsx

```diff
@@ -67,18 +69,29 @@ const Card = ({
               </span>
             </Tooltip>
           )}
-          {/* Project title and link */}
-          <Link href={url} target="_blank" rel="noopener noreferrer" className="flex-1">
-            <h1
-              className="max-w-full break-words text-base font-semibold dark:text-sky-600 sm:break-normal sm:text-lg lg:text-2xl"
-              style={{
-                transition: 'color 0.3s ease',
-              }}
-            >
-              {title}
-            </h1>
-          </Link>
+
+          <div className="flex flex-wrap items-center">
+            <Link href={url} target="_blank" rel="noopener noreferrer" className="inline-flex">
+              <h1
+                className="break-words text-base font-semibold dark:text-sky-600 sm:break-normal sm:text-lg lg:text-2xl"
+                style={{
+                  transition: 'color 0.3s ease',
+                }}
+              >
+                {title}
+                {labels?.map((label, index) => (
+                  <span
+                    key={`label-${index}`}
+                    className="mb-3 ml-2 rounded-full bg-blue-100 px-2 py-0.5 text-xs font-medium text-blue-800 dark:bg-blue-900 dark:text-blue-200"
```

Comment:
Can we also make them a bit larger? Also, we mostly use *sky* shades for links. Could you update the classes to work with that one instead?
```suggestion
                    className="mb-3 ml-2 rounded-full bg-blue-100 px-2 py-0.5 text-sm font-medium text-blue-800 dark:bg-blue-900 dark:text-blue-200"
```

Reply:
Yes working on it